### ISSUE TITLE
[Bug]When clicking the tray icon, the dropdown app is covered by MacOS menu bar

### ISSUE THREAD

**Issue by virejdasani** [MayGo/tockler#136]
![Screenshot 2021-06-05 at 11 10 10 PM](https://user-images.githubusercontent.com/67495678/120900609-4eb4f780-c653-11eb-8479-1773bccf6514.png)

**Comment by MayGo** [comment]
Just to confirm. You have "Automatically hide and show menu bar" enabled?
<img width="669" alt="Screenshot 2021-06-06 at 09 21 00" src="https://user-images.githubusercontent.com/1929299/120914734-b47f9e80-c6a8-11eb-83f3-219aa306ae59.png">

**Comment by virejdasani** [comment]
Yes I do but the menu bar doesn't hide when I open Tockler from the menu bar

**Comment by MayGo** [comment]
thanks, will fix this.

**Comment by MayGo** [comment]
fixed in [4.0.6](https://github.com/MayGo/tockler/releases/tag/v4.0.6)
